pkg/crc/cluster/cluster.go

```diff
@@ -138,6 +139,41 @@ func StopAndRemovePodsInVM(sshRunner *ssh.SSHRunner) error {
 	return errors.RetryAfter(2, stopAndRemovePods, 2*time.Second)
 }
 
+func AddProxyConfigToCluster(oc oc.OcConfig, proxy *network.ProxyConfig) error {
+	cmdArgs := []string{"patch", "proxy", "cluster", "-p",
+		fmt.Sprintf(`{"spec":{"httpProxy":"%s", "httpsProxy":"%s", "noProxy":"%s"}}`, proxy.HttpProxy, proxy.HttpsProxy, proxy.NoProxy),
+		"-n", "openshift-config", "--type", "merge"}
+
+	if err := waitForOpenshiftAPIServer(oc); err != nil {
+		return err
+	}
+	_, stderr, err := oc.RunOcCommand(cmdArgs...)
+	if err != nil {
```

Comment:
Nit: in the same function, it would be better to be consistent with error handling :)
```
if err := ...; err != nil {
}
```
VS
```
err := ...
if err != nil {
}
```